sable/Cargo.toml

```diff
@@ -20,7 +20,7 @@ sable_ext.workspace = true
 clap = "4.5.6"
 reqwest = "0.12.4"
 shellexpand = "3.1.0"
-glob = "0.3.1"
+wax = "0.6.0"
```

Comment:
why wax?

Reply:
its generally fancier and is easier to work with when you have multiple globs and stuff, so we can add support for that later on, also fancier iterator support, and opt-in miette errors (for the time when we'll have fancy errors)